src/components/Common/UI/Checkbox/Checkbox.stories.tsx

```diff
@@ -0,0 +1,80 @@
+import type { Story } from '@ladle/react'
+import { useState } from 'react'
+import { Checkbox } from './Checkbox'
+
+export const Default: Story = () => {
+  const [checked, setChecked] = useState(false)
+  return (
+    <Checkbox
+      label="Accept terms and conditions"
+      name="terms"
+      checked={checked}
+      onChange={event => {
+        setChecked(event.target.checked)
```

Comment:
I think another opportunity to use ladle actions here for a better demo experience.

Reply:
Yeah i agree! thinking since this has already gotten quite substantial in size i can create a dedicated follow up? that would be a ticket involving unifying the form wrapper and actions for ease of implementing field stories (since we're going to need to keep rolling these out)